src/CompanyService.Validation/Company/CreateCompanyRequestValidator.cs

```diff
@@ -9,19 +9,28 @@ public class CreateCompanyRequestValidator : AbstractValidator<CreateCompanyRequ
         public CreateCompanyRequestValidator()
         {
             RuleFor(request => request.PortalName)
-                .NotEmpty();
+                .NotEmpty()
+                .WithMessage("PortalName can't be empty");
```

Comment:
it is good that you added an error message, but please do not use the camel case in the message